esp/src/eclwatch/ShowIndividualPermissionsWidget.js

```diff
@@ -0,0 +1,228 @@
+/*##############################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+############################################################################## */
+define([
+    "dojo/_base/declare",
+    "dojo/_base/lang",
+    "dojo/i18n",
+    "dojo/i18n!./nls/hpcc",
+
+    "dijit/registry",
+    "dijit/form/CheckBox",
+
+    "dgrid/editor",
+
+    "hpcc/GridDetailsWidget",
+    "hpcc/ws_access",
+    "hpcc/ESPUtil",
+
+    "dijit/layout/BorderContainer",
+    "dijit/layout/TabContainer",
+    "dijit/layout/ContentPane"
```

Comment:
Are these 3 dijits needed?  If they are in GridDetailsWidget.js then they won't be...